tests/source/imports.rs

```diff
@@ -62,7 +62,7 @@ use foo::{baz, qux as bar};
 use ::foo;
 use ::foo::{Bar};
 use ::foo::{Bar, Baz};
-use ::{Foo};
+use ::{qux};
```

Comment:
This was changed due to an existing bug I discovered during this process that only surfaces when using the 2018 edition (we didn't see it before because the default edition was 2015).

I've opened https://github.com/rust-lang/rustfmt/issues/3943 to track that separate issue